ee/agent/reset.go

```diff
@@ -87,11 +99,14 @@ func DetectAndRemediateHardwareChange(ctx context.Context, k types.Knapsack) {
 		munemoChanged = valueChanged(ctx, k, slogger, currentTenantMunemo, hostDataKeyMunemo)
 	}
 
-	if serialChanged && hardwareUUIDChanged {
+	// note that machineGuid is only collected for windows. machineGuidChanged will only ever be true there
```

Comment:
```suggestion
	// note that machineGuid is only collected for windows. machineGuidChanged will only ever have a meaningful value for Windows
```

I originally interpreted "machineGuidChanged will only ever be true there" as "machineGuidChanged can never false on Windows" -- I think this update or similar avoids that interpretation 😅

Reply:
oh yep! good call ty